src/composables/useRelatedFlawTrackers.ts

```diff
@@ -142,27 +143,41 @@ export function useRelatedFlawTrackers(
     }
   });
 
-  // Logic for isLoadingTrackers could be moved to useSingleFlawTrackers as an improvement
   watch(affectsBySelectedFlawId, (newRelatedAffects: Record<string, ZodAffectType[]>) => {
+    updateMultiFlawTrackers(newRelatedAffects, false);
+  }, { immediate: true });
+
+  watch(relatedFlaws, () => {
+    selectedRelatedFlaws.value.forEach((selectedFlaw) => {
+      const index = relatedFlaws.value.findIndex(({ uuid }) => uuid === selectedFlaw.uuid);
+      if (index !== -1) {
+        selectedRelatedFlaws.value[index].affects = relatedFlaws.value[index].affects;
```

Comment:
This `index` comes from the `relatedFlaws` array, that holds all the related flaws, `selectedRelatedFlaws` is only populated by the ones selected so it will not always match

Reply:
Ah these are reversed!